lib/perl/Genome/Db/Ensembl/Api.pm

```diff
@@ -27,19 +27,21 @@ sub create {
     my @package_names = $self->package_names;
     #OLD CVS location
     #my $base_url = "'http://cvs.sanger.ac.uk/cgi-bin/viewvc.cgi/PACKAGENAME.tar.gz?root=ensembl&only_with_tag=branch-ensembl-VERSION&view=tar'";
-    my $base_url = "https://github.com/Ensembl/PACKAGENAME/archive/release/VERSION.zip";
+    #OLD github location
+    #my $base_url = "https://github.com/Ensembl/PACKAGENAME/archive/release/VERSION.zip";
```

Comment:
Good point - looks like they now have git branches for all legacy releases, so I removed this old code.  (It's always there in the history anyway!)